diff -rupN linux-3.12/drivers/misc/dte_test.c linux-3.12_modif/drivers/misc/dte_test.c
--- linux-3.12/drivers/misc/dte_test.c	1970-01-01 01:00:00.000000000 +0100
+++ linux-3.12_modif/drivers/misc/dte_test.c	2014-07-18 18:00:12.164594298 +0200
@@ -0,0 +1,14 @@
+#define __KERNEL__
+#define MODULE
+#include <linux/module.h>
+#include <linux/kernel.h>
+
+
+int init_module(void){
+	printk("bos días mundo \n");
+	return 0;
+}
+
+void cleanup_module(void){
+	printk("adeus mundo \n");
+}
diff -rupN linux-3.12/drivers/misc/Kconfig linux-3.12_modif/drivers/misc/Kconfig
--- linux-3.12/drivers/misc/Kconfig	2013-11-04 00:41:51.000000000 +0100
+++ linux-3.12_modif/drivers/misc/Kconfig	2014-07-18 17:38:17.744552344 +0200
@@ -40,6 +40,13 @@ config AD525X_DPOT_I2C
 	  To compile this driver as a module, choose M here: the
 	  module will be called ad525x_dpot-i2c.
 
+config DTE_TEST
+	tristate "driver dte"
+	depends on I2C
+	help
+	  diche se as porcas estan preparadas para darlles gherra. Se queren tubo, salta
+	un aviso de porca activada. GOOD BYE CRUEL WORLD!!!!!
+
 config AD525X_DPOT_SPI
 	tristate "support SPI bus connection"
 	depends on AD525X_DPOT && SPI_MASTER
diff -rupN linux-3.12/drivers/misc/Makefile linux-3.12_modif/drivers/misc/Makefile
--- linux-3.12/drivers/misc/Makefile	2013-11-04 00:41:51.000000000 +0100
+++ linux-3.12_modif/drivers/misc/Makefile	2014-07-18 17:14:33.980190923 +0200
@@ -4,6 +4,7 @@
 
 obj-$(CONFIG_IBM_ASM)		+= ibmasm/
 obj-$(CONFIG_AD525X_DPOT)	+= ad525x_dpot.o
+obj-$(CONFIG_DTE_TEST)		+= dte_test.o
 obj-$(CONFIG_AD525X_DPOT_I2C)	+= ad525x_dpot-i2c.o
 obj-$(CONFIG_AD525X_DPOT_SPI)	+= ad525x_dpot-spi.o
 obj-$(CONFIG_INTEL_MID_PTI)	+= pti.o
